strings/strings.ko-KR.json

```
@@ -2443,38 +2443,38 @@
   "tech_neutronium_housing_effect": "%0에서의 생활 공간 비용 감소",
   "tech_unification": "통일",
   "tech_unification_desc": "%0 통일",
-  "tech_unification_effect": "모든 국가 지배",
+  "tech_unification_effect": "전 세계를 당신의 통치 하에 통일할 계획을 세웁니다.",
   "tech_unification_effect2": "통일을 완료하려면 3개의 도시를 획득해야 합니다.",
   "tech_wc_conquest": "정복",
   "tech_wc_conquest_desc": "%0 정복",
   "tech_wc_conquest_desc_req": "모든 도시를 지배해야 합니다.",
   "tech_wc_conquest_effect": "강한 군사력으로 세계 정복",
   "tech_unification_warning": "통일 후에는 더 이상 전투가 없지만, 생산력은 25% 증가하고, 저장 용량은 2배 증가합니다.",
   "tech_unification_banana": "통일은 바나나 공화국 시나리오를 끝냅니다. 추가적인 바나나 스무디나 바나나 빙수 업적을 얻을 수 없게 될 것입니다.",
-  "tech_unite": "연합 국가",
-  "tech_unite_desc": "Unite your region of city states into a country.",
-  "tech_unite_effect": "Become one unified country, gain bonuses to storage and production.",
+  "tech_unite": "국가 통일",
+  "tech_unite_desc": "우리 지역에 있는 도시 국가들을 하나의 국가로 통일합니다.",
+  "tech_unite_effect": "하나의 통일된 국가가 됩니다. 저장 공간과 생산량 보너스를 얻습니다.",
   "tech_wc_morale": "문화 패권",
   "tech_wc_morale_desc": "%0 합병",
   "tech_wc_morale_desc_req": "행복도 %0% 필요",
-  "tech_wc_morale_effect": "높은 문화로 %0을/를 합병합니다.",
+  "tech_wc_morale_effect": "우리 국가의 우월한 문화로 %0을/를 점령합니다.",
   "tech_wc_money": "세계 구매",
   "tech_wc_money_desc": "%0 구매",
   "tech_wc_money_desc_req": "비용: $%0",
-  "tech_wc_money_effect": "%0을/를 구매합니다.",
+  "tech_wc_money_effect": "국가 전복 활동에 자금을 지원해 %0의 통제권 장악을 시도합니다.",
   "tech_wc_reject": "통일 거부",
-  "tech_wc_reject_effect": "세계 정복 취소",
+  "tech_wc_reject_effect": "세계 정복 계획을 포기하고 그 대신 우리 사회를 개선하는 데 집중합니다.",
   "tech_wc_reject_warning": "행복도 +20%",
   "tech_genesis": "제네시스 프로젝트",
-  "tech_genesis_effect": "우리 종족은 망한 것 같습니다. 그러니 다른 세계에 생명을 만들어서 정착시키는 계획을 세우세요.",
+  "tech_genesis_effect": "우리 종족은 망한 것 같습니다. 그러니 우리는 또 다른 항성계에 있는 행성에 우리가 설계한 생명을 파종시켜 지속적인 유산을 남길 수 있는 가능성을 연구해야 합니다.",
   "tech_star_dock": "우주 조선소",
-  "tech_star_dock_effect": "고급 우주선을 제작할 수 있는 우주 조선시설 설계",
+  "tech_star_dock_effect": "최첨단 우주선을 건조할 수 있는 우주 기반 조선시설 설계",
   "tech_interstellar": "성간 구동기",
   "tech_interstellar_effect": "실험용 엔진을 탑재하기 위해 특별히 설계된 우주선이 필요하지만, 수백 년이 아닌 수십 년 안에 인근의 다른 항성계에 도달할 수 있는 새로운 설계의 성간 구동기입니다.",
   "tech_genesis_ship": "제네시스 우주선",
   "tech_genesis_ship_effect": "먼 행성에 생명을 뿌리기 위한 우주선",
   "tech_generational_ship": "탈출 우주선",
-  "tech_generational_effect": "다른 행성으로 탈출하기 위한 우주선",
+  "tech_generational_effect": "먼 항성계의 새로운 고향으로 우리 종족을 수송하는 데 사용할 탈출선 설계",
```

```suggestion
  "tech_generational_effect": "먼 항성계를 새로운 고향으로 삼게 될 우리가 만든 종족을 수송할 탈출선 설계",
```
앞쪽에서 종족이 이동하는게 아니라 새로 만든 종족을 보내는 것이라고 하니 조금 손보면 어떨까 싶네요